src/api/customer/customer-controller.ts

```diff
@@ -15,6 +16,38 @@ const createCustomer = (): CustomerCreationRequestHandler[] => {
     ]
 }
 
+const findOneCustomer = (): CustomerFindOneRequestHandler[] => {
+    return [
+        getCustomerValidator(),
+        getCustomerById,
+        findCustomerSerializer,
+        (req, res) => {
+            if(res.locals.customerToRespond !== undefined){
```

Comment:
Mesma coisa, poderia apenas testar se é um `falsy value`.

```suggestion
            if (res.locals.customerToRespond) {
```